MAVProxy/modules/mavproxy_messagerate.py

```diff
@@ -76,10 +106,19 @@ def idle_task(self):
 
     def mavlink_packet(self, m):
         '''handle mavlink packets'''
+        self.sysid = (m.get_srcSystem(),m.get_srcComponent())
         mtype = m.get_type()
         if mtype not in self.counts:
             self.counts[mtype] = 0
         self.counts[mtype] += 1
+        
+        mavlink_map = mavutil.mavlink.mavlink_map
+        if mtype ==  'MESSAGE_INTERVAL':
+          if  m.message_id in mavlink_map:
+              print("Msg:%s interval %dus" % (mavlink_map[m.message_id].name, m.interval_us))
+          else:
+              print("Msg ID:%s interval %dus" % (m.message_id, m.interval_us))
```

Comment:
Is it possible for the ap to send a message id that is not included in the defintions in pymavlink?  This would not invalidate the checksum or message integrity.